src/CMakeLists.txt

```diff
@@ -172,8 +177,38 @@ install(FILES
   DESTINATION include/libcellml/module
 )
 
-install(EXPORT libcellml-targets DESTINATION lib/cmake)
+include(CMakePackageConfigHelpers)
+set(LIBCELLML_CONFIG_VERSION_FILE "${PROJECT_BINARY_DIR}/libcellml/libcellml-config-version.cmake")
+write_basic_package_version_file(
+  ${LIBCELLML_CONFIG_VERSION_FILE}
+  VERSION ${libCellML_VERSION}
+  COMPATIBILITY AnyNewerVersion
+)
+list(APPEND CLEAN_FILES ${LIBCELLML_CONFIG_VERSION_FILE})
+
+set(LIBCELLML_TARGETS_FILE "${PROJECT_BINARY_DIR}/libcellml/libcellml-targets.cmake")
+export(EXPORT libcellml-targets FILE ${LIBCELLML_TARGETS_FILE})
+list(APPEND CLEAN_FILES ${LIBCELLML_TARGETS_FILE})
+
+set(LIBCELLML_CONFIG_CMAKE_FILE "${PROJECT_BINARY_DIR}/libcellml/libcellml-config.cmake")
+configure_file("${PROJECT_SOURCE_DIR}/cmake/libcellml-config.cmake"
+  ${LIBCELLML_CONFIG_CMAKE_FILE}
+  COPYONLY
+)
+list(APPEND CLEAN_FILES ${LIBCELLML_CONFIG_CMAKE_FILE})
+
+set(ConfigPackageLocation lib/cmake/libcellml)
+install(EXPORT libcellml-targets
+    FILE libcellml-targets.cmake
+    DESTINATION ${ConfigPackageLocation})
```

Comment:
- 4-space indentation vs. 2-space above (see line 195 for example)?
- Have the closing bracket at the beginning of a new line (see line 197 for example).